data/json/mapgen/house/house_detatched6.json

```diff
@@ -57,13 +57,15 @@
         "{": "t_reinforced_door_glass_c",
         "}": "t_reinforced_glass_shutter",
         ")": "t_wall_glass",
-        "]": "t_door_metal_pickable"
+        "]": "t_door_metal_pickable",
+        "§": "t_region_groundcover_urban"
       },
-      "furniture": { "?": "f_beaded_door", "!": "f_region_flower", ",": "f_pinball_machine" },
+      "furniture": { "?": "f_beaded_door", "!": "f_region_flower", ",": "f_pinball_machine", "§": "f_wooden_flagpole" },
       "place_item": [
         { "item": "television", "repeat": 1, "x": 4, "y": 4 },
         { "item": "cannabis", "repeat": 1, "x": 18, "y": 14 },
-        { "item": "cannabis", "repeat": 1, "x": 18, "y": 15 }
+        { "item": "cannabis", "repeat": 1, "x": 18, "y": 15 },
+        { "item": "pride_flag", "chance": 50, "x": 12, "y": 16 }
```

Comment:
Is that 50% of spawn with pride flag there? Maybe make it a bit lower and/or put the other flags in the rotation

Reply:
Thought that would fit the grower home, while state and US flag do not so much I think. Pride flags come in variants, but I am not sure how to add the other base flag types. I guess it requires an item group? Or can it be done here directly?

Reply:
Done. Mixed in 20% US and 10% state flags.